qp_klp/AmpliconStep.py

```diff
@@ -0,0 +1,102 @@
+from os import listdir, makedirs
+from os.path import join, isfile, basename
+from shutil import copyfile
+from qp_klp.Step import Step
+
+
+class AmpliconStep(Step):
+    def __init__(self, pipeline, master_qiita_job_id, sn_tid_map_by_project,
+                 status_update_callback=None):
+        super().__init__(pipeline,
+                         master_qiita_job_id,
+                         sn_tid_map_by_project,
+                         status_update_callback)
+
+        if pipeline.pipeline_type != 'amplicon':
+            raise ValueError("Cannot instantiate AmpliconStep object from "
```

Comment:
Should this just be Amplicon? The AmpliconStep description might be too technical, what do you think?

Reply:
Your call. It's more wordy but I think it was convention for so long to name subclasses like this that I got used to it. Amplicon would be nice and shorter too; not sure if it will get confusing with other usages of the word 'amplicon' in the code. What do you propose?

Reply:
Just to be clear, I'm talking about the ValueError message, my concern is that a user will not know the difference between AmpliconStep/Amplicon and might confuse them, what do you think?

Reply:
I see where you're coming from but it could be argued that Error messages are as much for the developer as the user. The stack-traces we show in Qiita to the user have little value to the user, but convey that an error occurred and provide valuable insight to the developer.